yt_dlp/extractor/reddit.py

```diff
@@ -147,19 +144,42 @@ def add_thumbnail(src):
                 for resolution in resolutions:
                     add_thumbnail(resolution)
 
-        return {
-            '_type': 'url_transparent',
+        info = {
+            'id': video_id,
             'url': video_url,
             'title': data.get('title'),
             'thumbnails': thumbnails,
             'timestamp': float_or_none(data.get('created_utc')),
             'uploader': data.get('author'),
-            'duration': int_or_none(try_get(
-                data,
-                (lambda x: x['media']['reddit_video']['duration'],
-                 lambda x: x['secure_media']['reddit_video']['duration']))),
```

Comment:
This wasn't removed, just moved. Since this only is valid when the media is hosted on reddit (i.e., has `reddit_video`)
```python
reddit_video = try_get(data, lambda x: x['media']['reddit_video'])
if reddit_video:
    ...
```

Reply:
However, I did forget to add `secure_media` as a fallback to `media`. Will fix that.

Reply:
I meant the `secure_media` part
```py
                 lambda x: x['secure_media']['reddit_video']['duration']
```

Reply:
Fixed 👍